mozilla/bedrock
# [fix bug 1109132] Implement Firefox Hello FTUE

This has already been tested by both Firefox and Web QA pretty heavily on demo2, but still needs code review. If you're testing using 35 beta, please make sure it is up to date. All of the features used here are also in the latest Nightly.

To test locally, open `about:config`

1.) set `loop.gettingStarted.url` = `http://127.0.0.1:8000/%LOCALE%/firefox/%VERSION%/hello/start/`
2.) set `loop.gettingstarted.seen` = `false`
3.) whitelist your local server to use UiTour API by creating a pref called `browser.uitour.testingOrigins` and give it a value of `http://127.0.0.1:8000`
4.) restart the browser
5.) Click the hello icon in the browser to open the Hello panel.

You should now see a blue button that says “Get Started”. Clicking this will open the tour and if all the above steps have worked you should see the FTE page on demo and the first door-hanger pointing at “Start a conversation”

Note if you want to retake the tour without resetting `loop.gettingstarted.seen`, the tour is also available via the ‘Tour’ link in the Hello panel gear menu
